### ISSUE TITLE
"unexpected keyword argument 'wait'" when connecting to a XMPP MUC

### ISSUE THREAD

**Issue by raspbeguy** [errbotio/errbot#1506]
### I am...

* [x] Reporting a bug
* [ ] Suggesting a new feature
* [ ] Requesting help with running my bot
* [ ] Requesting help writing plugins
* [ ] Here about something else

### I am running...

* Errbot version: 6.1.7
* OS version: Debian 10
* Python version: 3.7
* Using a virtual environment: yes

### Issue description

Unable to join a rooms, seems to be a syntax issue, maybe related to migration to slixmpp.
Here is the backtrace:

```
2021-02-18 16:06:20,864 ERROR    errbot.plugins.ChatRoom   Joining room 'hashtagueule@conference.gugod.fr' failed
Traceback (most recent call last):
  File "/opt/bot/venv/lib/python3.7/site-packages/errbot/core_plugins/chatRoom.py", line 20, in callback_connect
    self._join_room(room)
  File "/opt/bot/venv/lib/python3.7/site-packages/errbot/core_plugins/chatRoom.py", line 37, in _join_room
    username=self.bot_config.CHATROOM_FN, password=password
  File "/opt/bot/venv/lib/python3.7/site-packages/errbot/backends/xmpp.py", line 136, in join
    self.xep0045.join_muc(room, username, password=password, wait=True)
TypeError: join_muc() got an unexpected keyword argument 'wait'
```

When I edit xmpp.py and remove this argument `wait`, it works.

### Steps to reproduce

Just set up a new bot with a XMPP backend.

**Comment by nzlosh** [comment]
The underlying Python module was switched from sleekxmpp to slixmpp in mid-2020.  It's possible the `join_muc` signature was overlooked during that work.  If the fix works, a PR would be most welcome.

**Comment by raspbeguy** [comment]
fixed by #1507
